src/main/java/com/jcabi/log/MulticolorLayout.java

```diff
@@ -75,7 +75,7 @@
  *  &lt;groupId&gt;com.jcabi&lt;/groupId&gt;
  *  &lt;artifactId&gt;jcabi-log&lt;/artifactId&gt;
  * &lt;/dependency&gt;</pre>
- *
+ * @todo #59:30min Continue refactoring of the class started in #51.
```

Comment:
@piddubnyi can you provide more details here? Like what was already done and what is still left?